docs/data/core-blocks.md

```diff
@@ -0,0 +1,86 @@
+# **core/blocks**: Block Types Data
+
+## Selectors 
+
+### getBlockType
+
+Returns a block type by name.
+
+*Parameters*
+
+ * state: Data state.
+ * name: Block type name.
+
+
+### getCategories
+
+Returns all the available categories.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Categories list.
+
+### getDefaultBlockName
+
+Returns the name of the default block name.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Default block name.
+
+### getFallbackBlockName
+
+Returns the name of the fallback block name.
```

Comment:
A lot of these functions are somewhat clear based on their function names, so an explanation like this one doesn't really help, I think.

But the docs could really shine if this explained (not in detail, but even just in passing), what a fallback block was. Why you'd use this and what it's for would really help contextualise a lot of these selector functions.